docs/faq.md

```diff
@@ -0,0 +1,77 @@
+
+# Tool Kit FAQs
+
+Below are responses to some general questions about Tool Kit that have been asked during migrations. If your question is not answered here please ask in the [#tool-kit](https://app.slack.com/client/T025C95MN/C02TRE2V2Q1/thread/C042NBBTM-1671107986.307219) slack channel and consider adding it, and the answer, to this document.
+
+### How do I add package specific configuration (for e.g. ESLint) to the Tool Kit plugin (e.g. @dotcom-tool-kit/eslint))?
+
+As much as possible Tool Kit doesn't handle configuration for third party packages. The purpose of Tool Kit’s plugin for packages is to get them working in the FT development workflow with the minimum possible configuration. Tool Kit plugins will leave package configuration to a specific configuration file for that package (e.g. `.eslintrc` for the ESLint package).
+
+### How do I run watch mode for tests with Tool Kit?
+
+Now that we manage tasks with npm scripts, you can define a script to run your tests in watch mode:
+
+```json
+{ "scripts": 
+    {
+        "watch-tests": "jest --watch" 
+    } 
+}
+```
+Then run `npm watch-tests` to execute your script.
+
+You can also use the package directly with npx to run your test runner with any options you wish:
+
+`npx jest path/to/tests --watch`
```

Comment:
**nitpick:** We should wrap this in a codeblock which will make this line easier to copy.

```suggestion
```sh
npx jest path/to/tests --watch
```.
```

(delete `.` from the above suggestion, it was the only way to show the markdown codeblock within a code block).